frontend/src/global_styles/content/work_packages/tabs/_files.sass

```diff
@@ -41,9 +41,13 @@
           opacity: 0.5
 
         &.disabled
```

Comment:
these should really be modifiers instead of extra classes, but I guess that's a TODO for a different PR